packages/page-staking/src/Actions/Account/ListNominees.tsx

```diff
@@ -18,6 +20,39 @@ function ListNominees ({ nominating, stashId }: Props): React.ReactElement<Props
   const { t } = useTranslation();
   const { api } = useApi();
   const { nomsActive, nomsChilled, nomsInactive, nomsOver, nomsWaiting } = useInactives(stashId, nominating);
+  const sessionInfo = useCall<DeriveSessionProgress>(api.query.staking && api.derive.session?.progress);
+  const [nomBalanceMap, setNomBalanceMap] = useState<Record<string, BN>>({});
+
+  useEffect(() => {
+    let unsub: (() => void) | undefined;
+
+    if (nomsActive && sessionInfo?.activeEra) {
+      api.derive.staking.eraExposure(sessionInfo.activeEra, (v) => {
```

Comment:
Would prefer just using a `useCall` with a transform. (Since we don't need to manage state, as it stands this also doesn't take care of unmounts, which useCall does without worries)

Reply:
Probably not with a transfers - useCall to retrieve the data and then a useMemo to combine it. (Since otherwise the active noms will be captured in the closure, so the memo will take both the era and noms in to get a result)

But yes, quite heavy atm since it subscribes/unsubscribes all the time.